pkg/clusterapi/system.go

```diff
@@ -244,6 +255,7 @@ func (c *system) Run(ctx context.Context) error { //nolint:gocyclo
 					"AZURE_SUBSCRIPTION_ID":             session.Credentials.SubscriptionID,
 					"AZURE_RESOURCE_MANAGER_ENDPOINT":   session.Environment.ResourceManagerEndpoint,
 					"AZURE_RESOURCE_MANAGER_AUDIENCE":   session.Environment.TokenAudience,
+					"AZURE_ENVIRONMENT_FILEPATH":        ashFN, // TODO: not sure if needed on this controller or just CAPZ
```

Comment:
clean up comment

Reply:
still a TODO? Looks like its needed in both place?